src/Template.php

```diff
@@ -52,24 +52,20 @@ public function __construct(Environment $env)
 
     /**
      * Returns the template name.
-     *
-     * @return string The template name
      */
-    abstract public function getTemplateName();
+    abstract public function getTemplateName() : string;
```

Comment:
```suggestion
    abstract public function getTemplateName(): string;
```

Reply:
How come this is not automatically enforced by PHP CS Fixer or fabbot? I feel your valuable time can be used for other things 😁